Libraries/Microsoft.Teams.Apps/Routing/Router.cs

```diff
@@ -24,6 +24,7 @@ public class Router : IRouter
     public IList<IRoute> Select(IActivity activity)
     {
         return _routes
+            .OrderBy(route => route.Type == RouteType.User ? 0 : 1)
```

Comment:
so user routes run before system? We should make sure this is in other languages too